components/org.wso2.carbon.analytics.auth.rest.api/src/main/java/org/wso2/carbon/analytics/auth/rest/api/impl/LogoutApiServiceImpl.java

```diff
@@ -116,4 +123,27 @@ public Response logoutAppNamePost(String appName
         errorDTO.setDescription("Invalid Authorization header. Please provide the Authorization header to proceed.");
         return Response.status(Response.Status.BAD_REQUEST).entity(errorDTO).build();
     }
+
+    @Override
```

Comment:
We shouldn't be specific for ExternalIdPClient. If a custom IdPClient is used, this will break the flow.

Reply:
+1 on this. Will fix this in a separate PR as an improvement, since this is not a blocking issue.